front/packages/models/src/resources/watch-info.ts

```diff
@@ -29,9 +29,42 @@ const getDisplayName = (sub: Track) => {
 	if (lng && sub.title && sub.title !== lng) return `${lng} - ${sub.title}`;
 	if (lng) return lng;
 	if (sub.title) return sub.title;
-	return `Unknwon (${sub.index})`;
+	return `Unknown (${sub.index})`;
 };
 
+/**
+ * A Video track
+ */
+export const VideoP = z.object({
+	/**
+	 * The Codec of the Video Track.
+	 * E.g. "AVC"
+	 */
+	codec: z.string(),
+	/**
+	 * The Quality of the Video
+	 * E.g. "1080p" (TODO: FIND ACTUAL ENUM)
```

Comment:
The enum is actually not typed in the front, I added it inline here https://github.com/zoriya/Kyoo/blob/cc9b6e69c83193abbad6332cca08a9f510d84938/front/packages/models/src/resources/user.ts#L42 (not on master), it could be nice to extract it into another const.